scipy/stats/stats.py

```diff
@@ -1039,6 +1095,16 @@ def moment(a, moment=1, axis=0):
 
     """
     a, axis = _chk_asarray(a, axis)
+
+    contains_nan, nan_policy = _contains_nan(a, nan_policy)
+
+    # TODO: implement the delegation to mstats when nan_policy == 'omit'
+    # if contains_nan and nan_policy == 'omit':
+    #     a = ma.masked_invalid(a)
```

Comment:
This is commented until we come up with a way to deal with the problem importing `mstats`.